packages/extension-ui/src/Popup/CreateAccount/AccountCredentials.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019-2020 @polkadot/extension-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React, { useRef, useState } from 'react';
+import { Address, Button, ButtonArea } from '@polkadot/extension-ui/components';
+import { Name, Password, DerivationPath } from '@polkadot/extension-ui/partials';
+import { OnDerivationPathChangeProps } from '@polkadot/extension-ui/partials/DerivationPath';
+import { KeypairType } from '@polkadot/util-crypto/types';
+import styled from 'styled-components';
+
+interface Props {
+  onCreate: (name: string, password: string, suri: string, type?: KeypairType) => void | Promise<void | boolean>;
+  seed: string;
+}
+
+export default function AccountCredentials ({ onCreate, seed }: Props): React.ReactElement<Props> {
+  const [name, setName] = useState<string | null>(null);
+  const [password, setPassword] = useState<string | null>(null);
+  const [address, setAddress] = useState<string | null>(null);
+  const [suri, setSuri] = useState<string | null>(null);
+  const [pairType, setPairType] = useState<KeypairType | undefined>(undefined);
+
+  const ref = useRef<HTMLDivElement | null>(null);
+
+  const scroll = (way: 'up' | 'down'): void => {
+    return ref?.current?.scrollTo({
+      top: way === 'up' ? -10000000 : 100000000, // a sufficiently large number
+      behavior: 'smooth'
+    });
+  };
```

Comment:
```suggestion
  const scroll = useCallback((way: 'up' | 'down'): void => {
    return ref?.current?.scrollTo({
      top: way === 'up' ? -10000000 : 100000000, // a sufficiently large number
      behavior: 'smooth'
    });
  }, []);
```

I know you are just following in the path I set out, but this is actually something we need to start doing. So at least when we start new components (or touch old), let's start doing thing "the-right-way(tm)" :)